testsuite/features/secondary/srv_maintenance_windows.feature

```diff
@@ -0,0 +1,142 @@
+    And I click on "Add Calendar"
+    And I select "multicalendar" from "calendarSelect"
+    And I click on "Create Schedule"
+    Then I should see a "Schedule successfully created" text
+
+  Scenario: Assign a single system to the single schedule
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Properties" in the content area
+    And I select "singleschedule" from "maintenance-schedule"
+    And I click on "Update Properties"
+    Then I should see a "System properties changed" text
+
+  Scenario: Assign systems to a multi schedule using SSM
+    When I follow the left menu "Systems > Overview"
+    And I follow "Clear"
+    And I check the "sle_client" client
+    And I follow the left menu "Systems > System Set Manager > Overview"
+    And I follow "Assign" in the content area
+    And I select "SAP Maintenance Window" from "scheduleId"
+    And I check "cancelActions"
+    And I click on "Assign All"
+    And I click on "Confirm"
+    Then I should see a "Maintenance schedule has been assigned" text
+
+  Scenario: Schedule channel change action
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    Then radio button "Test-Channel-x86_64" is checked
+    When I wait until I do not see "Loading..." text
+    Then I should see "Test-Channel-x86_64 Child Channel" as unchecked
+    When I check "Test-Channel-x86_64 Child Channel"
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I pick "17:30" as time
+    And I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+
+  Scenario: Schedule package installation action
+    When I remove package "virgo-dummy" from this "sle_client" without error control
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Software" in the content area
+    And I follow "Install"
+    And I check "virgo-dummy" in the list
+    And I click on "Install Selected Packages"
+    And I select the next maintenance window
+    And I click on "Confirm"
+    Then I should see a "1 package install has been scheduled for" text
+
+  Scenario: Detach systems from schedules
+    When I follow the left menu "Systems > Overview"
+    And I follow "Clear"
+    And I check the "sle_client" client
+    And I check the "sle_minion" client
+    And I follow the left menu "Systems > System Set Manager > Overview"
+    And I follow "Assign" in the content area
+    And I select "None - clear schedule" from "scheduleId"
+    And I click on "Clear All"
+    Then I should see a "Maintenance schedule has been cleared" text
+
+  Scenario: Cleanup: cancel all scheduled actions
+    Given I am logged in API as user "admin" and password "admin"
+    When I cancel all scheduled actions
+    And I logout from API
```

Comment:
Why not encapsulate the authentication inside the step?

Reply:
As said above, let's define as a clear strategy to move all authentication to feature prerequisite and cleanup.